CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ database. Use the GitHub user interface or a text editor like emacs or NotePad i
 
 **Inclusion criteria**
 
-- [ ] Make sure that any faculty you add meet the inclusion
+- [X] Make sure that any faculty you add meet the inclusion
```

Comment:
This file should not be part of this PR, and needs to be removed (not "deleted", which would delete it from the repository).

Reply:
Hi Emery, I'm not quite sure how to do that -- could you help me here?

Reply:
I suggest you close this PR and open a new PR without `CONTRIBUTING.md`.

Reply:
I was able to remove the file from the PR.  Let me know if this works.  Thanks!